tern/__main__.py

```diff
@@ -216,6 +216,9 @@ def main():
                                "available formats: "
                                "spdxtagvalue, spdxjson, cyclonedxjson, json, "
                                "yaml, html")
+    parser_report.add_argument('-fv', '--format-version',
```

Comment:
since this is only for SPDX, suggest making that more clear in the wording. Maybe -sv for spdx-version? or just v? Also clarify in the `help` that it is only for the SPDX version and will otherwise be ignored.

Reply:
Also probably want to indicate what the valid input is. When I test with -fv 2.3 it doesn't work.

Reply:
Done, thanks for the input! :)